app/views/effective/classifieds/show.html.haml

```diff
@@ -30,21 +30,21 @@
               = link_to(@classified.file.filename, main_app.url_for(@classified.file), target: '_blank')
 
 
-        %ul.list-unstyled
-          - if @classified.website.present?
-            %li
-              %strong Website
-              = link_to(@classified.website, @classified.website, target: '_blank')
+        -# %ul.list-unstyled
```

Comment:
I know we have Git and no need to comment out code but if we want to add them back would be super quick. Can remove it if we decide is better

Reply:
Yea, let's delete this commented out code please